Mage.Sets/src/mage/cards/l/LifecraftEngine.java

```diff
@@ -0,0 +1,100 @@
+package mage.cards.l;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AsEntersBattlefieldAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.common.ChooseCreatureTypeEffect;
+import mage.abilities.effects.common.continuous.BoostAllOfChosenSubtypeEffect;
+import mage.abilities.keyword.CrewAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ *
+ * @author jackd149
+ */
+public final class LifecraftEngine extends CardImpl {
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent();
```

Comment:
must be `filter = StaticFilters.FILTER_CONTROLLED_CREATURE` or `filter = new FilterControlledCreaturePermanent` due "Each creature you **control** of the chosen type"